src/components/pages/profile/file-input.tsx

```diff
@@ -0,0 +1,114 @@
+'use client'
+
+import ImageCard from '@/components/pages/auth/image-card'
+import { Icons } from '@/components/ui/icons'
+import { Progress } from '@/components/ui/progress'
+import { cn } from '@/lib/utils'
+import { FC, FormEvent, useRef, useState } from 'react'
+
+type FileInputProps = {
+  className?: string
+}
+
+const FileInput: FC<FileInputProps> = ({ className }) => {
+  const fileInputRef = useRef<HTMLInputElement>(null)
+
+  const [_, setImage] = useState<File | undefined>()
+
```

Comment:
Remove unnecessary empty lines. 

Suggesting always compose our components as this (see line breaks): 

```tsx
const Component = () => {
  // states
  
  // callbacks
  
  // effects
  
  // const needed for rendering
  return </>;
}
```

Reply:
Did I miss smth, but can u show me, what exactly I wrote wrong? 'Cause I can't find out, what's wrong, code writed according to your suggested pattern. Or did useRef responds to "const needed for rendering"?(`useRef is a React Hook that lets you reference a value that’s not needed for rendering.`)

Reply:
I mean there are too many blank lines

![image](https://github.com/dev-KPI/messenger-frontend/assets/95881717/91cdf875-103e-4666-a581-506012f4265c)

Reply:
I do not mind structuring code with blank lines, but seems we should not separate each line with blank line.

Reply:
It 'seems' to u, but first answer pls https://github.com/dev-KPI/messenger-frontend/pull/94#discussion_r1469709691

Reply:
Btw if u have nowhere to spend your free time (except to bother others with shit things) you can play in games e.g.

Reply:
Personally I prefer to avoid using empty lines where it is not nesessary. If I wrote this code, I would remove empty lines just between react hooks, and thats it